src/workflow/StreamlitUI.py

```diff
@@ -31,9 +32,10 @@ def __init__(self, workflow_dir, logger, executor, paramter_manager):
     def upload_widget(
         self,
         key: str,
-        file_type: str,
+        file_types: Union[str, List[str]],
         name: str = "",
         fallback: Union[List, str] = None,
+        symlink: bool = True,
```

Comment:
Symlinks introduce a source of error for developers as files can be changed at anytime and this has to be accounted for when developing WebApps. Thus, I would argue this is an advanced feature and should not be enabled by default.

Reply:
That's a good point